scripts/variantstore/wdl/extract/utils.py

```diff
@@ -53,3 +53,22 @@ def execute_with_retry(client, label, sql):
 
 def utf8len(s):
     return len(s.encode('utf-8'))
+
+
+def dump_job_stats(jobs, client, fq_dataset, call_set_identifier, step, call, shard_identifier):
```

Comment:
perhaps this should be `write_job_stats` since this is being persisted to the db as well now?